gravitee-apim-console-webui/src/management/api/creation-v4/steps/step-2-entrypoints/step-2-entrypoints-0-architecture.component.scss

```diff
@@ -37,3 +37,7 @@ $typography: map.get(gio.$mat-theme, typography);
     margin: 0;
   }
 }
+
+gio-banner-error {
```

Comment:
What do you think about adding a dedicated class to add this style? like `license__banner`
We're trying to respect "css bem"